<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Convert ASCII Code</string>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>331</width>
     <height>221</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="inputLabel">
      <property name="text">
       <string>Plain Text:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="outputLabel">
      <property name="text">
       <string>ASCII Code:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPlainTextEdit" name="outputTextBox">
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="plainText">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLineEdit" name="inputTextBox">
        <property name="inputMask">
         <string/>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="placeholderText">
         <string>Try to type somethings...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="toolButton">
        <property name="toolTip">
         <string>Remove all</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="application.qrc">
          <normaloff>:/images/delete</normaloff>:/images/delete</iconset>
        </property>
        <property name="arrowType">
         <enum>Qt::NoArrow</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>260</y>
     <width>201</width>
     <height>26</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="changeButton">
      <property name="toolTip">
       <string>Click to change mode</string>
      </property>
      <property name="text">
       <string>Change</string>
      </property>
      <property name="icon">
       <iconset resource="application.qrc">
        <normaloff>:/images/2arrow</normaloff>:/images/2arrow</iconset>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="convertButton">
      <property name="toolTip">
       <string>Click to convert</string>
      </property>
      <property name="text">
       <string>Convert</string>
      </property>
      <property name="icon">
       <iconset resource="application.qrc">
        <normaloff>:/images/convert</normaloff>:/images/convert</iconset>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>inputTextBox</tabstop>
  <tabstop>outputTextBox</tabstop>
  <tabstop>changeButton</tabstop>
  <tabstop>convertButton</tabstop>
  <tabstop>toolButton</tabstop>
 </tabstops>
 <resources>
  <include location="application.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>toolButton</sender>
   <signal>clicked()</signal>
   <receiver>inputTextBox</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>348</x>
     <y>44</y>
    </hint>
    <hint type="destinationlabel">
     <x>293</x>
     <y>41</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolButton</sender>
   <signal>clicked()</signal>
   <receiver>outputTextBox</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>347</x>
     <y>48</y>
    </hint>
    <hint type="destinationlabel">
     <x>347</x>
     <y>86</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>inputTextBox</sender>
   <signal>editingFinished()</signal>
   <receiver>Widget</receiver>
   <slot>convert()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>155</x>
     <y>45</y>
    </hint>
    <hint type="destinationlabel">
     <x>182</x>
     <y>21</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>convertButton</sender>
   <signal>clicked()</signal>
   <receiver>Widget</receiver>
   <slot>convert()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>293</x>
     <y>275</y>
    </hint>
    <hint type="destinationlabel">
     <x>309</x>
     <y>295</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>changeButton</sender>
   <signal>clicked()</signal>
   <receiver>Widget</receiver>
   <slot>changeType()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>216</x>
     <y>276</y>
    </hint>
    <hint type="destinationlabel">
     <x>222</x>
     <y>292</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>convert()</slot>
  <slot>changeType()</slot>
 </slots>
</ui>
